custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -396,6 +396,4 @@ def indexes(self):
         return [
             'CREATE INDEX IF NOT EXISTS chm_case_idx ON "{}" (case_id)'.format(self.tablename),
             'CREATE INDEX IF NOT EXISTS chm_awc_idx ON "{}" (awc_id)'.format(self.tablename),
-            'CREATE INDEX IF NOT EXISTS chm_mother_dob ON "{}" (mother_case_id, dob)'.format(self.tablename),
```

Comment:
I think this was likely added for https://github.com/dimagi/commcare-hq/blob/3d5f09c80e2ccc0caac3361b36b05d2cfae086ff/custom/icds_reports/utils/aggregation_helpers/distributed/mbt.py#L244-L245

but this couldn't be used because all child_health_monthly rows are already in memory, also for similar reasons as above, the supervisor, month filter is much better as the table grows